webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -5090,6 +5201,12 @@ const firstVisibleMsg = {
                     '#mck-conversation-back-btn',
                     function (e) {
                         e.preventDefault();
+                        const businessHourBox = document.getElementById(
+                            'km-business-hour-box'
+                        );
+                        if (businessHourBox) {
+                            businessHourBox.classList.add('n-vis');
+                        }
```

Comment:
No, we are not removing element from DOM. I generally add a check while fetching DOM node to avoid runtime errors for any unhandled scenario. 
I have removed the check and manipulating it directly.